Mathlib/NumberTheory/NumberField/InfinitePlace/TotallyRealComplex.lean

```diff
@@ -62,6 +78,55 @@ instance : IsTotallyReal ℚ where
     rw [Subsingleton.elim v Rat.infinitePlace]
     exact Rat.isReal_infinitePlace
 
+section maximalRealSubfield
+
+open ComplexEmbedding
+
+/--
+The maximal real subfield of `K`. It is totally real, see `isTotallyReal_maximalRealSubfield`, and
```

Comment:
Is `isTotallyReal_maximalRealSubfield` in `_root`? I am sorry for not checking myself, I am not at VSCode, but if not the doc HTML won't work as expected. Also, why not mentioning at least part of this doc at the top of the file?

Reply:
Good point. I added the namespace to the docstring of the definition and completed the one at the top of the file. Thanks!

Reply:
Thanks!